confd/pkg/backends/calico/routes_test.go

```diff
@@ -626,3 +630,21 @@ var _ = Describe("RouteGenerator", func() {
 		})
 	})
 })
+
+var _ = Describe("Update BGP Config Cache", func() {
+	c := &client{cache: make(map[string]string)}
+
+	It("should update cache value when IgnoredInterfaces set in BGPConfiguration", func() {
```

Comment:
nit: "should update cache value when IgnoredInterfaces is set in BGPConfiguration"